napari-workshops/notebooks/manual_annotation.md

```diff
@@ -45,34 +44,33 @@ from napari.utils import nbscreenshot
 viewer = napari.Viewer()
 ```
 
-In this notebook we will load our data directly into napari using out builtin plugin reader
+In this notebook we will use the same `cells3d` data set:
```

Comment:
```suggestion
In this notebook we will use the `cells3d` data set:
```

Reply:
it's the same data, just loaded a different way, so maybe we leave `same`?

Reply:
I kinda wonder if we should just switch everything to use the skimage data so people dont need the repo cloned with the data file. I think showing how to open a data file is important though for intro.

Reply:
Maybe open an issue with that suggestion, and we can follow up on it